Microbiology Society Journals.js

```diff
@@ -0,0 +1,236 @@
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    Zotero is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+    GNU Affero General Public License for more details.
+
+    You should have received a copy of the GNU Affero General Public License
+    along with Zotero. If not, see <http://www.gnu.org/licenses/>.
+
+    ***** END LICENSE BLOCK *****
+*/
+
+function detectWeb(doc, url) {
+    if ( url.search(/\/content\/journal\/\w+\/10\./) >-1) {
+    	//e.g. http://jmm.microbiologyresearch.org/content/journal/jmm/10.1099/00222615-15-2-189
+        return "journalArticle";
+    }
+    if (getSearchResults(doc, true)) {
+    	//e.g. http://ijs.microbiologyresearch.org/content/journal/ijsem/65/11
+    	//http://jmm.microbiologyresearch.org/search?option1=author&noRedirect=true&value1=Ateequr+Rehman&facetOptions=2&facetNames=pub_author_facet&operator2=AND&option2=pub_author_facet&value2=%27Patricia+Lepage%27
+        return "multiple";
+    }
+}
+
+function getSearchResults(doc, checkOnly) {
+    var items = {};
+    var found = false;
+    var rows = ZU.xpath(doc, '//li[contains(@class, "list-group-item")]//h4/a');
+    for (var i=0; i<rows.length; i++) {
+        var href = rows[i].href;
+        var title = ZU.trimInternal(rows[i].textContent);
+        if (!href || !title) continue;
+        if (checkOnly) return true;
+        found = true;
+        items[href] = title;
+    }
+    return found ? items : false;
+}
+
+
+function doWeb(doc, url) {
+    if (detectWeb(doc, url) == "multiple") {
+        Zotero.selectItems(getSearchResults(doc, false), function (items) {
+            if (!items) {
+                return true;
+            }
+            var articles = new Array();
+            for (var i in items) {
+                articles.push(i);
+            }
+            ZU.processDocuments(articles, scrape);
+        });
+    } else {
+        scrape(doc, url);
+    }
+}
+
+function scrape(doc, url) {
+	var risURL;
+	var node = ZU.xpath(doc, '//ul[@id="export-list"]/li/a[contains(@title,"RefWorks")]');
+	if (node.length > 0) {
+		risURL = node[0].href;
+	} else {
+		risURL = url + "/cite/refworks";
+	}
+
+    ZU.doGet(risURL, function(text) {
+        var translator = Zotero.loadTranslator("import");
+        translator.setTranslator("1a3506da-a303-4b0a-a1cd-f216e6138d86");
+        translator.setString(text);
+        translator.setHandler("itemDone", function(obj, item) {
+            var pdfURL;
+		    pdfURL = ZU.xpath(doc, '//div[contains(@class,"textoptionsFulltext")]//a[contains(@class,"pdf")]');
+            if (pdfURL.length > 0) {
+                item.attachments.push({
+                    url: pdfURL[0].href,
+                    title: "Full Text PDF",
```

Comment:
I don't have a super strong opinion on this, but I don't think I agree. Do you recall why? Yes, we want the generic label in there (i.e. they should all contain "Snapshot/Full Text PDF") for searchability, but why would we not want to see where they come from? That's actually one of the principal reasons for me to want a generic label rather than simply using the item.title, which some users expect/prefer.